webserver/Controllers/RealEstatesController.cs

```diff
@@ -64,60 +61,51 @@ public class RealEstatesController : ControllerBase {
         }
 
         if(offset.HasValue){
-            realEstates=realEstates.Skip(offset.Value);
+            realEstates = realEstates.Skip(offset.Value);
         }
         if(limit.HasValue){
-            realEstates=realEstates.Take(limit.Value);
+            realEstates = realEstates.Take(limit.Value);
         }
 
         var resultArray = await realEstates.ToArrayAsync();
-        var resultDtoArray = realEstates.Select(r=>(RealEstateDTO)r).ToArray();
 
-        if(resultDtoArray==null){
+        if(resultArray==null || resultArray.Length==0){
```

Comment:
Why would this function return null?
Do you have a test case for that?

One test case returning null, other returning length zero*

Reply:
I just read the Docs and turns out `realEstates.ToArrayAsync();` does not return null
So i removed the `resultArray==null`

With that in mind, the tests do cover this endpoint